packages/gatsby/src/utils/api-node-docs.ts

```diff
@@ -431,6 +431,7 @@ export const onCreateDevServer = true
 
 /**
  * Called during `gatsby develop` bootstrap to get and validate a plugins options schema
+ * @gatsbyVersion 2.24.69
```

Comment:
Note that this is going to need to be bumped to whatever version is the first to include the `.default()`s support from #27381 (which isn't merged yet).

Reply:
Oh wow, this is an incredible feature! Thanks for pointing the PR, will study on that 🚀